src/test/java/com/puppycrawl/tools/checkstyle/filters/SuppressionPatchXpathFilterTest.java

```diff
@@ -122,4 +122,21 @@ public void testMemberName() throws Exception {
         testByConfig(
                 "MemberName/patchedline/defaultContextConfig.xml");
     }
+
+    @Test
+    @Ignore("MethodLength should have a violation when method length"
```

Comment:
please put "until <LINK to issue>", it is much easier to see where are we have problems.